curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/handle-a-rejected-promise-with-catch.english.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 5cdafbe72913098997531682
+title: Handle a Rejected Promise with catch
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+<code>catch</code> is the method used when your promise has been rejected. It is executed immediately after a promise's <code>reject</code> method is called. Here’s the syntax:
+
+```js
+myPromise.catch(error => {
+  // do something with the error.
+});
+```
+
+<code>error</code> is the argument passed in to the <code>reject</code> method.
+
+<strong>Note:</strong> the <code>then</code> and <code>catch</code> methods can be chained to the promise declaration if you chose.
+</section>
+
+## Instructions
+<section id='instructions'>
+Add the <code>catch</code> method to your promise and log its argument to the console.
+</section>
+
+## Tests
+<section id='tests'>
+
+```yml
+tests:
+  - text: You should properly use the <code>catch</code> method.
+    testString: assert(code.match(/(makeServerRequest|\))\.catch\(\s*(\s*function\s*\(\s*error\s*\)\s*|\s*error\s*=>\s*|\s*\(\s*error\s*\)\s*=>\s*){[^}]*}\s*\)/g));
+  - text: You should log the <code>catch</code> argument to the console.
+    testString: assert(code.match(/{[^c]*console\.log\(\s*error\s*\)[^}]*}/g));
+```
+
+</section>
+
+## Challenge Seed
+<section id='challengeSeed'>
+<div id='js-seed'>
+
+```js
+const makeServerRequest = new Promise((resolve, reject) => {
+  const responseFromServer = true;
```

Comment:
```suggestion
  let responseFromServer;
  \\ the server responds and responseFromServer is set to true or false
```
Mirroring the change to the other challenge.

Reply:
```suggestion
\\ responseFromServer is set to false to represent an unsuccessful response from a server
let responseFromServer = false;
```

Reply:
Seems good!